### DISCUSSION TITLE
woowacourse/real-time-data-transport/

### DISCUSSION THREAD

**Discussion by giscus** [rbgksqkr/rbgksqkr.github.io#2] [Announcements]
# woowacourse/real-time-data-transport/

땅콩 서비스를 개발하면서 실시간 데이터 통신 방식에 대해 고민하고, 기술 결정에 대한 이유를 소개한다.

https://rbgksqkr.github.io/woowacourse/real-time-data-transport/

**Comment by chlwlstlf** [comment]
그림까지 그렸네여ㅋㅋ👍
